src/shared/constants/index.ts

```diff
@@ -38,6 +38,8 @@ export const Constants = {
   AWS_HA_MASTERS_VERSION: '11.4.0',
   AWS_HA_MASTERS_MAX_NODES: 3,
 
+  AZURE_CAPZ_VERSION: '20.0.0',
```

Comment:
We only fetch the new `v1alpha4` CRs if the cluster release version is `>=20.0.0`